tools/checkswitch/testdata/testdata.go

```diff
@@ -16,9 +16,8 @@
 package testdata
 
 import (
-	"time"
```

Comment:
I guess that was changed accidentally. 😃

Reply:
Oh, that's my IDE :/ I turned off all the autoformatting but it keeps changing this file :/